crt_portal/cts_forms/templates/forms/widgets/usa_checkbox_option.html

```diff
@@ -6,5 +7,5 @@
     class="usa-checkbox__input"
     {% include "django/forms/widgets/attrs.html" %}
   />
-  <label role="label" class="crt-checkbox__label" {% if widget.attrs.id %} for="{{ widget.attrs.id }}"{% endif %}>{{ widget.label }}</label>
+  <label role="label" class="crt-checkbox__label" {% if widget.attrs.id %} for="{{ widget.attrs.id }}"{% endif %}>{% trans widget.label %}</label>
```

Comment:
Labels should also be served with lazy loading, these should be covered.

Reply:
Addressed in #437